src/platform/Zephyr/PlatformManagerImpl.cpp

```diff
@@ -76,33 +76,42 @@ static int app_entropy_source(void * data, unsigned char * output, size_t len, s
 
 void PlatformManagerImpl::OperationalHoursSavingTimerEventHandler(k_timer * timer)
 {
-    PlatformMgr().ScheduleWork(UpdateOperationalHours);
+    PlatformMgr().ScheduleWork([](intptr_t arg) {
+        CHIP_ERROR error = sInstance.UpdateOperationalHours(nullptr);
+        if (error != CHIP_NO_ERROR)
+        {
+            ChipLogError(DeviceLayer, "Failed to update operational hours: %s", chip::ErrorStr(error));
+        }
+    });
 }
 
-void PlatformManagerImpl::UpdateOperationalHours(intptr_t arg)
+CHIP_ERROR PlatformManagerImpl::UpdateOperationalHours(uint32_t * aTotalOperationalHours)
```

Comment:
nit: Matter codebase tries to avoid `aArg` notation

Reply:
changed `totalOperationalHours` variable to `totalTime` and `aTotalOperationalHours` argument to `totalOperationalHours`